common/src/app/common/types/tokens_lib.cljc

```diff
@@ -665,20 +665,20 @@
      ["$value" :map]
      ["$type" :string]]]))
 
-(defn has-legacy-format?
+(defn get-json-format
```

Comment:
I've changed the predicate to a function that returns a file format keyword, since relying on the distinction between `true`, `false`, and `nil` in a language with nil punning seems error-prone.